### ISSUE TITLE
Script gets rid of "Sort" button and the filter "Video" "Short" and "All" buttons in watch later playlist

### ISSUE THREAD

**Issue by Enju93** [adamlui/userscripts#108]
![image_2024-10-29_200454845](https://github.com/user-attachments/assets/c567eace-7cf1-43a5-b7f0-5aa4d88cc2ec)
![image](https://github.com/user-attachments/assets/8869e347-022b-41bf-90bf-1e4241829541)

I've tested this with disabling all other Scripts/ Extension I'm running on youtube and this is what triggers this to happen somehow.

*Edit* Also gets rid of them on all other playlists as well

**Comment by adamlui** [comment]
@Enju93 thanks for the report, my UI is older than yours so doen'st have those extra buttons so I can't tell what their properties are to find out what CSS selector is hiding it so I took a guess, can you update and see if it worked?

**Comment by Enju93** [comment]
Updated the script and tested, buttons still hidden.

Not sure if the element where the buttons should be is missing orbuttons themselves are just missing
![image_2024-10-30_164329530](https://github.com/user-attachments/assets/671619c7-329f-417c-b4e2-99d8bdbfc715)

**Comment by adamlui** [comment]
Can you copy/paste this into console and hit enter

```js
ytd-item-section-renderer.style-scope.ytd-section-list-renderer[page-subtype="playlist"] > #header.ytd-item-section-renderer > ytd-feed-filter-chip-bar-renderer
```

Does it return null or does it highlight that bar? Also what usersccript manager and browser are you using so I can try to emulate

**Comment by Enju93** [comment]
I'm using Opera GX with Tampermonkey.  

Tried that code with and without the script, with the script it won't higlight even the plalist element, but without the script it will highlight the container with ther buttons that are missing

**Comment by Enju93** [comment]
And with the text you sent me I found the problem, there was a code which disabled the said container, delete that part and voila buttons are back.

**Comment by adamlui** [comment]
Yes that is where I copied it from thanks to your useful pic! I will delete it in GitHub code after smoke break so your userscript manager will auto update to future necessary changes
